tools/sigclient/pkg/query/query.go

```diff
@@ -909,3 +919,175 @@ func RunQueryFromFileAndOutputResponseTimes(dest string, filepath string, queryR
 
 	log.Infof("RunQueryFromFileAndOutputResponseTimes: Query results written to CSV file: %v", queryResultFile)
 }
+
+var skipIndexes = map[int]bool{
+	3:   true,
+	34:  true,
+	213: true,
+	219: true,
+	220: true,
+	221: true,
+	222: true,
+	223: true,
+	224: true,
+	225: true,
+	226: true,
+	227: true,
+	228: true,
+	229: true,
+	374: true,
+	375: true,
+	376: true,
+	377: true,
+	290: true,
+	292: true,
+	294: true,
+	295: true,
+	296: true,
+	297: true,
+	312: true,
+	313: true,
+	314: true,
+	315: true,
+	316: true,
+	317: true,
+	318: true,
+	319: true,
+	320: true,
+	321: true,
+	322: true,
+	323: true,
+	324: true,
+	325: true,
+	326: true,
+	327: true,
+	328: true,
+	329: true,
+	330: true,
+	331: true,
+	332: true,
+	333: true,
+	334: true,
+	335: true,
+	336: true,
+	337: true,
+	338: true,
+	339: true,
+	340: true,
+	341: true,
+	342: true,
+	343: true,
+	344: true,
+	345: true,
+	346: true,
+	347: true,
+	348: true,
+	349: true,
+	350: true,
+	351: true,
+	352: true,
+	353: true,
+	354: true,
+	355: true,
+	356: true,
+	357: true,
+	358: true,
+	359: true,
+	360: true,
+	361: true,
+	362: true,
+	363: true,
+	364: true,
+	365: true,
+	366: true,
+	367: true,
+	368: true,
+	369: true,
+	370: true,
+	371: true,
+	372: true,
+	373: true,
```

Comment:
nit: could you arrange the above numbers in a sequence, so that it is easier to follow

Reply:
also add a comment at each line quickly why it is not working, for e.g. 
```
// mv cmds not implemented
// eval-stats issue
// etc.
```